src/Umbraco.Configuration/Models/UserPasswordConfigurationSettings.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+using Umbraco.Core.Configuration;
+
+namespace Umbraco.Configuration.Models
+{
+    internal class UserPasswordConfigurationSettings : IUserPasswordConfiguration
+    {
+        private readonly IConfiguration _configuration;
+        public UserPasswordConfigurationSettings(IConfiguration configuration)
+        {
+            _configuration = configuration;
+        }
+
+        public int RequiredLength => _configuration.GetValue<int?>("Umbraco:CMS:Security:UserPassword:RequiredLength") ?? 10;
+        public bool RequireNonLetterOrDigit =>  _configuration.GetValue<bool?>("Umbraco:CMS:Security:UserPassword:RequireNonLetterOrDigit") ?? false;
+        public bool RequireDigit => _configuration.GetValue<bool?>("Umbraco:CMS:Security:UserPassword:RequireDigit") ?? false;
```

Comment:
Side note, MemberPasswordConfigurationSettings very similar to this now, noted for the user/membership work